cl/people_db/views.py

```diff
@@ -80,6 +80,24 @@ def view_person(request, pk, slug):
     }
     oral_arguments_heard = conn.query().add_extra(**q).execute()
     conn.conn.http_connection.close()
+    conn = ExtraSolrInterface(settings.SOLR_RECAP_URL, mode="r")
+    q = {
```

Comment:
You'll need to use the grouping parameters to make this work properly. That means removing the collapse query parser and instead using the grouping parameters. Something like:

```
            {
                "group": "true",
                "group.ngroups": "true",
                "group.limit": 1,  # We only want one result per case
                "group.field": "docket_id",
                "group.sort": "dateFiled desc,
            },
```

Reply:
See `search_utils.add_grouping` for an example.

Reply:
I think we're gonna test out this method in production.  if im not incorrect.

Reply:
this should be resolved.